azure-quantum/azure/quantum/qiskit/backends/rigetti.py

```diff
@@ -53,7 +55,10 @@ def __init__(
 
     @classmethod
     def _default_options(cls):
-        return Options(count=500, targetCapability="BasicExecution")
+        other_options = {
+            cls._SHOTS_PARAM_NAME: 500
```

Comment:
we've been setting the default options through `_DEFAULT_SHOTS_COUNT` earlier, we should probably follow the same pattern here.

Reply:
Yeah, I just thought it may be ok to keep it as literal since it is currently used only in one place, but I guess it wouldn't harm 🙂